src/rofl/common/crofconn.cc

```diff
@@ -276,8 +267,7 @@ crofconn::set_state(
 		}
 
 	} catch (std::runtime_error& e) {
-
-		journal.log(LOG_RUNTIME_ERROR, "runtime error: %s", e.what()).set_caller(__PRETTY_FUNCTION__);
+		VLOG(1) << __PRETTY_FUNCTION__ << " runtime error: " << e.what();
```

Comment:
in principle yes, but rofl by default never logged anything. That's why I just used the VLOGs. Basically ERROR -> VLOG(1), INFO -> VLOG(2) and TRACE -> VLOG(3) (should have added this to the commit message).